pages/teach.md

```diff
@@ -0,0 +1,10 @@
+---
+permalink: /teach/
```

Comment:
This is a link in the "TEACH" menu item but currently gives a 404

Reply:
Filed #28 to address.

Reply:
Oh, this *is* the fix. Shoulda gone straight to the code view instead of reading the comments thread.